torchrl/modules/distributions/continuous.py

```diff
@@ -322,36 +324,58 @@ class TanhNormal(FasterTransformedDistribution):
 
     num_params = 2
 
+    def _warn_minmax(self):
+        warnings.warn(
+            f"the min / high keyword arguments are deprecated in favor of low / high in {type(self).__name__} "
+            f"and will be removed entirely in v0.6. ",
+            DeprecationWarning,
+        )
+
     def __init__(
         self,
         loc: torch.Tensor,
         scale: torch.Tensor,
         upscale: Union[torch.Tensor, Number] = 5.0,
-        min: Union[torch.Tensor, Number] = -1.0,
-        max: Union[torch.Tensor, Number] = 1.0,
-        event_dims: int = 1,
+        low: Union[torch.Tensor, Number] = -1.0,
```

Comment:
If we change this we should change it in all bounded distributions. I would do it in a separate PR